### ISSUE TITLE
Build a general training tool for supervised training

### ISSUE THREAD

**Issue by oysteijo** [oysteijo/simd_neuralnet#59]
There should be a general tool for loading training data and train a neural network supervised.
Say, call it `general-trainer` and it could take in a few command line options.

* Trainset - Filename.npz
* Verificationset - Filename.npz
* Neuralnet - filename.npz
* loss - string
* metrics - list of metrics

* Optimizer - String SGD, RMS prop, Adam or adamw
* Batch size - integer
* do_shuffle.

* Log - log_filename
* Early stopping - patience, idx, greater_is_better
* Model check point - filename, idx, greater_is_better, verbose

Then there could also be another tool that creates a new untrained neural network. Say, `create-neuralnet`. Just let this idea spin for a while,

**Comment by oysteijo** [comment]
OK - branch 'training_tool' has been created.

**Comment by oysteijo** [comment]
Pull request #65 created.
